src/Dockerfile

```diff
@@ -14,3 +14,7 @@ RUN helm init --client-only && \
     helm repo add rimusz https://charts.rimusz.net
 
 COPY --chown=circleci:circleci . /home/circleci/
+
+# Quick fix to avoid having to rebase all PRs.
+RUN echo "alias pip=pip3" >> /etc/bash.bashrc
+RUN echo "alias python=python3" >> /etc/bash.bashrc
```

Comment:
This files gets overridden during build, and it's being removed on #72.

`RUN` also adds extra layers to the docker container image, so maybe is simpler to just rebase PRs? :grin:

Reply:
Yeah I thought it would be a quick fix to avoid asking Stefan for a rebase, but yeah I could also ask him :smile: 
As for the layers, since it's at the end it shouldn't have a noticeable effect. The layers are really small, and don't invalidate any big layer.

Reply:
I added your commit in #72 in a `.bashrc` file, since we already copy files into `$HOME`. And it's good to have these aliases anyway for convenience.

Reply:
I added your commit in #72 in a `.bashrc` file, since we already copy files into `$HOME`. And it's good to have these aliases anyway for convenience.